OCast/Core/DeviceCenter.swift

```diff
@@ -71,6 +71,11 @@ public class DeviceCenter: NSObject, DeviceDiscoveryDelegate {
         set { deviceDiscovery?.interval = newValue }
     }
     
+    /// A Boolean value that indicates whether the discovery process is currently running.
```

Comment:
///  Indicates if the device discovery process is currently running.

Reply:
https://developer.apple.com/documentation/avfoundation/avaudioengine/1388591-isrunning

Reply:
So it's missing the *Discussion* section : "The value is true if the discovery process is running, otherwise, false." ;)